apps/web/src/components/reaction-button.tsx

```diff
@@ -23,7 +23,7 @@ export function ReactionButtons({ reactions, reactToKey }: ReactionButtonProps)
           <form key={key} action={reactToPage}>
             <Button
               type="submit"
-              className={`${reactions[Number(key)]?.hasReacted ? "bg-wave hover:bg-wave" : "bg-muted hover:bg-muted"} h-8 w-14 rounded-full text-foreground`}
+              className={`${reactions[Number(key)]?.hasReacted ? "bg-wave hover:bg-wave text-wave-foreground" : "bg-muted hover:bg-muted text-foreground"} h-8 w-14 rounded-full`}
```

Comment:
Kunne dette ha vært i `cn`?

Reply:
Hva er cn?

Reply:
En funksjon som kombinerer strenger. Praktisk for tailwindcss.

Her blir det,

```
className={cn("h-8 w-14 rounded-full", reactions[Number(key)]?.hasReacted ? "bg-wave hover:bg-wave text-wave-foreground" : "bg-muted hover:bg-muted text-foreground")}
```

https://renatopozzi.me/articles/create-react-components-with-tailwind-like-a-pro